invoker/core/pom.xml

```diff
@@ -52,17 +52,17 @@
     <dependency>
       <groupId>io.cloudevents</groupId>
       <artifactId>cloudevents-core</artifactId>
-      <version>2.0.0-milestone4</version>
+      <version>2.0.0.RC2</version>
```

Comment:
Any way to put the `2.0.0.RC2` value into a variable or possibly group some of these artifacts together so we don't have to make the same change in 3 different places each time we bump the version number?